client/indexer-db/src/models/proofs.rs

```diff
@@ -0,0 +1,55 @@
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+
+use crate::{schema::proofs, DbConnection};
+
+/// A single record table that holds the state/metadata of the indexer service.
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = proofs)]
+pub struct Proofs {
+    pub id: i32,
+    pub provider_id: String,
```

Comment:
What do you think of renaming it to `onchain_provider_id` so it's more clear that it is not a DB id?